tools/klee/main.cpp

```diff
@@ -1036,7 +1036,11 @@ createLibCWrapper(std::vector<std::unique_ptr<llvm::Module>> &modules,
   args.push_back(
       llvm::ConstantExpr::getBitCast(inModuleRefernce, ft->getParamType(0)));
   args.push_back(&*(stub->arg_begin())); // argc
+#if LLVM_VERSION_CODE >= LLVM_VERSION(5, 0)
+  args.push_back(&*(stub->arg_begin() + 1)); // argv
+#else
   args.push_back(&*(++stub->arg_begin())); // argv
+#endif
```

Comment:
Can we somehow get rid of the `ifdef`? Sth. like (haven't checked the types):
```
auto arg_it = stubs->arg_begin();
...
args.push_back(&*(++arg_it));
```

Reply:
Yep, that works.